README.md

```diff
@@ -58,15 +66,11 @@ Your framework will have a folder with it's name and have 2 endpoints:
 - **POST /foo/** (validates that the frontend can talk to backend)
 - **POST /bar/** (validates that the frontend can talk to backend)
 
-## Folders
+### Folders
 
-Each Server framework is in its own folder (named after the framework) and has its own readme. You can safely ignore anything in the frameworks that you are not using.
+Each Server framework is in its own folder (named after the framework) and has its own Readme. You can safely ignore anything in the frameworks that you are not using.
 
 - [Rails](./rails/README.md)
 - [Spring](./spring/README.md)
 - [Django](./django/README.md)
 - [Node](./node/README.md)
-
-## Prototype!
```

Comment:
I removed this bit as it confused me during the set-up and I don't feel it's relevant then